fiftyone/zoo/models/manifest-torch.json

```diff
@@ -215,7 +215,7 @@
                     "support": true
                 }
             },
-            "tags": ["segment-anything-2", "torch", "zero-shot"],
+            "tags": ["segment-anything", "torch", "zero-shot"],
```

Comment:
Okay with using `segment-anything` for the entire family of segment anything models? 

I believe the code snippets for SAM1 and SAM2-Image are identical, so the built docs should still be valid with this tag change.